.gitignore

```diff
@@ -1,11 +1,19 @@
+# Sample files
+examples/*.mp4
+examples/sample_data
```

Comment:
we shouldn't ignore those sample files, should we? @vinicvaz

Reply:
I think we can keep the .mp4 files from examples folder in ignore. We don't want to upload the mp4 files from demo to the repo. The sample_data folder I'll remove, I was using only for my local development